modules/ingestor/src/graph/sbom/mod.rs

```diff
@@ -186,7 +186,7 @@ impl Graph {
             .map(|sbom| SbomContext::new(self, sbom)))
     }
 
-    async fn locate_many_sboms<TX: AsRef<Transactional>>(
+    pub async fn locate_many_sboms<TX: AsRef<Transactional>>(
```

Comment:
Most of these fn's you are using from `ingestor` should actually end up in some `FooService` in the `fundamental` module.

Reply:
clarifying: this isn't your problem. But we should probably move them out of `ingestor` (or provide similar in `fundamental` such as `SbomService`) for usage by GraphQL stuff.

Reply:
Do you want to try to move to the relevant FooService (augmenting as necessary?) in this PR, or do that in a subsequent one?

Reply:
@bobmcwhirter, I'd rather do it in a follow-up PR. It'll give me time to catch-up with you/@jcrossley3/@ctron so I can grasp the Entity/Ingestor/Fundamental layers roles.